### PR TITLE
Adding managed environment

### LINKED ISSUE #311: Feature Request: Add Managed Environment to the methodology and the reference implementation
**Describe the solution you'd like**
* Include guidance for managed environments in landing zone context, in the design areas for Power Platform Landing Zones
* Include option to enable managed environments for all landing zones and env creation in the reference implementation

### PR DESCRIPTION
**Pull request by krnese**
## Summary of the Pull Request

* Adding support for Managed Environments (preview) in the reference implementation
* Updated design guidelines to incorporate Managed Environments considerations and recommendations

## PR Checklist
* [ ] Closes Issue #xxx
* [ ] Code of Conduct signed. If not, go to [Code of Conduct](https://github.com/microsoft/industry/blob/main/CODE_OF_CONDUCT.md).

### CODE REVIEW

**Review by uday31in — DISMISSED** [review]

File: foundations/powerPlatform/readme.md [hunk 1]
```diff
@@ -203,6 +203,9 @@ The following section describes the design considerations and the design recomme
 - Environments can be created with - or without Dynamics 365 application templates available in the tenant.
 - Dynamics 365 applications takes a dependency on environments with Dataverse provisioned.
 - Dataverse for Microsoft Teams can be created directly from the Teams client, which grants the creator Owner permission on the Environment.
+- Managed Environments requires that the users are licensed, as all apps and flows in a managed environment are premium.
+- To enable Managed Environments, you must be a global admin, Power Platform admin, or Dynamics 365 admin in Azure Active Directory.
+- In the Power Platform admin center, Environment Admins are not allowed to activate managed environments. Environment Admins are able to see the 'Enable Managed Environment' action in the environment details page.
```
Comment (akadrno) [inline foundations/powerPlatform/readme.md:208]: There is a backlog item to address the issue of Environment Admins are able to see the 'Enable Managed Environment' action in the environment details page. Once Completed the "view" should be trimmed to only Role Admins.

File: foundations/powerPlatform/readme.md [hunk 2]
```diff
@@ -222,6 +225,10 @@ The following section describes the design considerations and the design recomme
 - Enable auditing for your tenant and environments to understand usage and available capacity.
 - For Microsoft Cloud for industry solutions (e.g., D365 applications for Healthcare, Financial Services Industry), deploy all Dynamics 365 applications to the same environment(s), and avoid creating islands of data that will be complex to merge and combine later.
 - Only split Microsoft Cloud for industry solutions (D365 applications) across different environments if there are data or security constraints.
+- Use Managed Environments for the environments as it provides a suite of capabilities that allows admins to manage Power Platform at scale with more control, such as:
+  - Weekly digest, which informs about what's happening in the environment, such as analytics about your top apps and flows, your must impactful makes, and inactive resources you can safely clean up, deliverd to one or more mailboxes once a week.
+  - Sharing limits, where admins can limit how broadly makers can share canvas apps, by either excluding the sharing capability entirely, or define the limits of how many the apps can be shared with.
```
Comment (akadrno) [inline foundations/powerPlatform/readme.md:230]: Excluding the sharing capability entirely can currently be accomplished by setting the sharing restriction value to "1".  Was the intent here to state this or was it to state that excluding group-based sharing capability is possible. Either way we should align to the current documents: https://docs.microsoft.com/en-us/power-platform/admin/managed-environment-enable

Exclude sharing with security groups | Blocks the ability to share canvas apps with any security groups. Admins may share with a limit on who an app can be shared with

Limit total individuals who can be shared to | If Exclude sharing with security groups is selected, admin my limit the number of users a canvas app is shared with

File: foundations/powerPlatform/readme.md [hunk 3]
```diff
@@ -203,6 +203,9 @@ The following section describes the design considerations and the design recomme
 - Environments can be created with - or without Dynamics 365 application templates available in the tenant.
 - Dynamics 365 applications takes a dependency on environments with Dataverse provisioned.
 - Dataverse for Microsoft Teams can be created directly from the Teams client, which grants the creator Owner permission on the Environment.
+- Managed Environments requires that the users are licensed, as all apps and flows in a managed environment are premium.
```
Comment (akadrno) [inline foundations/powerPlatform/readme.md:206]: Reword: Enabling Managed Environments promotes all recourses within an environment to premium. Users who access premium resources must be licensed with a premium license: https://docs.microsoft.com/en-us/power-platform/admin/signup-for-powerapps-admin

**Review by akadrno — COMMENTED** [review]
Comments added for review

**Review by daltondhcp — APPROVED** [review]
🚢

**Review by akadrno — APPROVED** [review]

**Pull request merged**